apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -157,19 +152,25 @@ type StagedUpdateRunStatus struct {
 	// +kubebuilder:validation:Optional
 	PolicySnapshotIndexUsed string `json:"policySnapshotIndexUsed,omitempty"`
 
+	// PolicyObservedNodeCount records the number of observed nodes in the policy snapshot.
```

Comment:
Hi Ryan! Would this be more accurate if we use cluster count instead of node count?